docs/classic-ui/theming/color-mode.md

```diff
@@ -149,4 +149,4 @@ See the following example.
 
 ## Using color modes in Plone 6.0
 
-There is also the possibility to use the color modes in Plone 6.0 by manually include `barceloneta=3.1.0` and `plone.staticresources=2.1.0`.
+To use color modes in Plone 6.0, manually include `barceloneta=3.1.0` and `plone.staticresources=2.1.0`, as described in {ref}`pin-the-version-of-an-add-on`.
```

Comment:
My apologies, I gave you the incorrect reference. This time I looked it up in the source.

```suggestion
To use color modes in Plone 6.0, manually include `barceloneta=3.1.0` and `plone.staticresources=2.1.0`, as described in {ref}`manage-add-an-add-on`.
```